src/renderer/cluster-modals/cluster-modals.tsx

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import styles from "./cluster-modals.module.css";
+
+import { withInjectables } from "@ogre-tools/injectable-react";
+import React from "react";
+import type { ClusterModalRegistration } from "../../extensions/registries";
+import clusterModalsInjectable from "./cluster-modals.injectable";
+
+interface Dependencies {
+  clusterModals: ClusterModalRegistration[];
+}
+
+export const NonInjectedClusterModals = ({ clusterModals }: Dependencies) => {
+  return (
+    <div className={styles.clusterModals} style={{ height: 0 }}>
+      {clusterModals.map((modal) => {
+        return modal.visible ? <modal.Component key={modal.id} /> : null;
```

Comment:
This is wrong, since `visible` is `IComputedValue<boolean>`

Reply:
It would be great if we had unit tests to verify this behaviour too

Reply:
Computed value now respected. Added test case with variable `visible` property.